src-docs/src/views/text_utilities/align.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { EuiCode, EuiMark } from '../../../../src';
+import { ThemeExample } from '../theme/_components/_theme_example';
+
+export default () => (
+  <>
+    <ThemeExample
+      title={<code>.eui-textLeft</code>}
+      description={
+        <p>
+          Changes the element’s text alignment property to{' '}
+          <EuiCode language="sass">text-align: left;</EuiCode>
```

Comment:
I just realized we need to be changing these all to logical properties. 😬

Reply:
I would still keep the name of the thing as `left` and `right`, but the output should be logical.

Reply:
As in I should be changing this in Sass itself? Could we maybe wait until we convert our Sass-only text utils to CSS-in-JS globals before doing this?

Reply:
Sorry no, I didn't realize this PR didn't contain the JS version of the Sass utility. When we do convert the pure utility, we need to make sure output uses the logical properties. Then in this description, instead of explicitly saying what the output style is, we'd just say something like:

```suggestion
          be aligned to the left (starting) side of its container.
```

Reply:
I made that documentation tweak while here since I figured it was easy enough - LMK if you'd rather revert/wait until we tackle Sass/Emotion globals https://github.com/elastic/eui/pull/5854/commits/b4d2c2d0c94adfcaaad62e69e3ba33a596a981d5